pkg/arvo/app/s3-store.hoon

```diff
@@ -89,7 +89,17 @@
   --
 ::
 ++  on-leave  on-leave:def
-++  on-peek   on-peek:def
+++  on-peek
+  ~/  %s3-peek
+  |=  =path
+  ^-  (unit (unit cage))
+  ?>  (team:title our.bowl src.bowl)
+  ?+    path  (on-watch:def path)
+      [%x %credentials ~]
+    [~ ~ %s3-update !>(`update`[%credentials credentials])]
+      [%x %configuration ~]
```

Comment:
I like to put a little extra emphasis on the separation between `?+` cases.

```suggestion
    [~ ~ %s3-update !>(`update`[%credentials credentials])]
  ::
      [%x %configuration ~]
```

Also for less-wide one-liners you could consider ``` ``s3-update+!>(etc)```, but it's fine either way.